app/src/atoms/Banner/__tests__/Banner.test.tsx

```diff
@@ -73,4 +73,14 @@ describe('Banner', () => {
     getByLabelText('icon_warning')
     getByText('TITLE')
   })
+  it('renders custom close', () => {
+    props = {
+      type: 'warning',
+      children: 'TITLE',
+      close: 'close button',
+      onCloseClick: jest.fn(),
+    }
+    const { getByText } = render(props)
+    getByText('close button')
```

Comment:
adding to this case or create a new case?

```suggestion
    const button = getByText('close button')
    fireEvent.click(button)
    expect(props.onCloseClick).toHaveBeenCalled()
```